t/cfg-args.t

```diff
@@ -80,6 +80,13 @@ push @tests, {
     'closed_path'    => 'cfg/*/',
     'expected_files' => 'single-uptodate.fr.po  single-uptodate.pot single-uptodate.man.fr.1',
 
+  },
+  {
+    'doc'            => '--target-lang',
+    'po4a.conf'      => 'cfg/args-global/po4a.conf',
+    'options'        => '--no-update --target-lang=de --target-lang=fr',
+    'expected_outfile' => '_output-target-lang',
+    'expected_files' => 'man.de.1 man.fr.1'
```

Comment:
Also, alignment.

```suggestion
    'doc'              => '--target-lang',
    'po4a.conf'        => 'cfg/args-global/po4a.conf',
    'options'          => '--no-update --target-lang=de --target-lang=fr',
    'expected_outfile' => '_output-target-lang',
    'expected_files'   => 'man.de.1 man.fr.1'
```

PS: I also mentioned before about testing if it affects the translation generation as well (i.e. dropping `--no-update`), but after some reflection I'm not so sure whether it actually will be beneficial.

Reply:
Hello, thank you for this review.

For the test, I tried to remove the "--no-update" option but that led to lots of other errors which I did not fully understood, due to the test fixture not really adapted what needs to be achieved. I also think this would not be very beneficial.

Reply:
Yeah... you are right... I've looked into it and it it would require adding a pot file, but it would mess up all the other tests of the same directory. It doesn't worth the effort.